bundles/org.openhab.binding.tapocontrol/README.md

```diff
@@ -0,0 +1,114 @@
+
+* Light On/Off
+* Brightnes (Dimmer)  0-100 %
+* ColorTemperature (Number) 2500-6500 K
+* Color (Color)
+* Wi-Fi signal (SignalStrength)
+* On-Time (Time in seconds device is switched on)
+
+
+## Prerequisites
+
+Before using Smart Plugs with openHAB the devices must be connected to the Wi-Fi network.
+This can be done using the Tapo provided mobile app.
+You need to setup a bridge (Cloud-Login) to commiunicate with your devices.
+
+## Discovery
+
+Discovery is done by connecting to the Tapo-Cloud Service. 
+All devices stored in your cloud account will be detected even if they are not in your network.
+You need to know the IP-Adress of your device. This must be set manually in the thing configuration
+
+## Bridge Configuration
+
+The bridge needs to be configured with by `username` and `password` (Tapo-Cloud login) .
+This is used for device discovery and to create a handshake (cookie) to act with your devices over the local network.
+
+The thing has the following configuration parameters:
+
+| Parameter          | Description                                                          |
+|--------------------|----------------------------------------------------------------------|
+| username           | Username (eMail) of your Tapo-Cloud                                  |
+| password           | Password of your Tapo-Cloud                                          |
+
+## Thing Configuration
+
+The thing needs to be configured with `ipAddress`.
+
+The thing has the following configuration parameters:
+
+| Parameter          | Description                                                          |
+|--------------------|----------------------------------------------------------------------|
+| ipAddress          | IP Address of the device.                                            |
+| pollingInterval    | Refresh interval in seconds. Optional. The default is 30 seconds     |
+
+
+## Channels
+
+All devices support some of the following channels:
+
+| group     | channel          |type                    | description                  | things supporting this channel  |
+|-----------|----------------- |------------------------|------------------------------|---------------------------------|
+| actuator  | output           | Switch                 | Power device on or off       | P100, P105,L510, L530, L900     |
+|           | brightness       | Dimmer                 | Brightness 0-100%            | L510, L530, L900                |
+|           | colorTemperature | Number                 | White-Color-Temp 2500-6500K  | L510, L530, L900                |
+|           | color            | Color                  | Color                        | L530, L900                       |
+| device    | wifiSignal       | system.signal-strength | WiFi-quality-level           | P100, P105, L510, L530, L900    |
+|           | onTime           | QuantityType <Time>    | seconds output is on         | P100, P105, L510, L530, L900    |
```

Comment:
```suggestion
|           | onTime           | Number:Time    | seconds output is on         | P100, P105, L510, L530, L900    |
```

Reply:
Summary Report is clear. Do you mean this warnings?
![image](https://user-images.githubusercontent.com/40909464/142777972-dccd13bd-9a69-4566-a457-ead1ff1041ca.png)
Fixing these will take a while... :/

Reply:
Yes.

Reply:
Good Morning. Updated ReadMe and fixed compiler warnings.